CORPORATE_SETUP.md

```diff
@@ -0,0 +1,111 @@
+
+# Setup Guide for Mac Users with Corporate git credentials.
```

Comment:
Maybe top heading is Setup Guide for multiple git credentials?
and then second level is Mac
and then continue on
That way we have a space for Windows eventually

Reply:
sounds good